.github/workflows/build-container.yaml

```diff
@@ -0,0 +1,65 @@
+name: Building and publishing container
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - "v*.*.*"
+      - "force-docker-build"
```

Comment:
Using the `force-docker-build` tag to trigger the workflow for PRs seems a little bit impractical to me. We would have to remember to explicitly delete the tag in order to be able to reuse it. This could be rather annoying if we want to use the tag on multiple PRs that are open at the same time.

We can keep this as it is, but a more viable alternative would be to add the `workflow_dispatch` event to `build-container.yaml`. This enables us to manually trigger the workflow on any branch. See [here](https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/) for an example (note: this example provides inputs for the workflow but we don't need any)